<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="0">
     <widget class="QPushButton" name="previewButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Camera Preview</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>300</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="16" colspan="2">
     <widget class="CamPreviewArea" name="camFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>380</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="projectName_label">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>None Loaded</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="refreshCamerasButton">
      <property name="text">
       <string>Refresh Camera Connection</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Project:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>40</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>250</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSetup">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <widget class="QPushButton" name="openProjectButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Open Project</string>
        </property>
       </widget>
       <widget class="QPushButton" name="editProjectSettingsButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Edit Project Settings</string>
        </property>
       </widget>
       <widget class="QPushButton" name="calibrateButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Calibrate Cameras</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camConfigButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
This is for loading new GenAPI configurations to set up your
cameras.&lt;br /&gt; &lt;br /&gt; 
 For example, if you want to change the shutter speed,&lt;br /&gt; or the lighting configuration, from what is currently loaded. &lt;br /&gt;&lt;br /&gt; If the camera has just been attached to the system, it&lt;br /&gt; 
most likely has a default configuration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Load Camera Config</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createProjectButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Create Project</string>
        </property>
       </widget>
       <widget class="QPushButton" name="getMaskButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
This is for loading new GenAPI configurations to set up your
cameras.&lt;br /&gt; &lt;br /&gt; 
 For example, if you want to change the shutter speed,&lt;br /&gt; or the lighting configuration, from what is currently loaded. &lt;br /&gt;&lt;br /&gt; If the camera has just been attached to the system, it&lt;br /&gt; 
most likely has a default configuration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Collect Image Mask</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRecording">
       <attribute name="title">
        <string>Collect Data</string>
       </attribute>
       <widget class="QPushButton" name="previewButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
       <widget class="QPushButton" name="recordButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camConfigButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
This is for loading new GenAPI configurations to set up your
cameras.&lt;br /&gt; &lt;br /&gt; 
 For example, if you want to change the shutter speed,&lt;br /&gt; or the lighting configuration, from what is currently loaded. &lt;br /&gt;&lt;br /&gt; If the camera has just been attached to the system, it&lt;br /&gt; 
most likely has a default configuration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Load Camera Config</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabProcessing">
       <attribute name="title">
        <string>Process</string>
       </attribute>
       <widget class="QPushButton" name="trackingButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Triangulate Points</string>
        </property>
       </widget>
       <widget class="QPushButton" name="visualisationButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>View Triangulated Points</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_export">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Export Triangulated Points</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Program icon originally from https://icons8.com/ .</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CamPreviewArea</class>
   <extends>QFrame</extends>
   <header>CamPreviewArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
